Lib/fontTools/colorLib/builder.py

```diff
@@ -59,7 +63,72 @@
 ]
 _AffineInput = Union[_AffineTuple, ot.Affine2x3]
 
+
```

Comment:
many of the typing aliases above are no longer used since you removed the function that these were annotating. You can remove from line 52 to 64

Reply:
Done ty